docs/intro.txt

```diff
@@ -120,7 +121,7 @@ Alternatively, you can store your credentials in local pytket config using the :
 
 After saving your credentials you can access ``pytket-qiskit`` backend repeatedly without having to re-initialise your credentials.
 
-If you are a member of an IBM hub then you can add this information to ``set_ibmq_config`` as well.
+If you are a member of an IBM hub then you can add this information to :py:meth:`~pytket.extensions.qiskit.backends.config.set_ibmq_config` as well.
```

Comment:
Crossreferncing this way means the link to the function/class is generated in the html page.